src/backend/distributed/utils/node_metadata.c

```diff
@@ -263,6 +267,43 @@ get_shard_id_for_distribution_column(PG_FUNCTION_ARGS)
 
 
 /*
+ * For the time-being master_activate_node is a wrapper around
+ * ReplicateAllReferenceTablesToNode().
```

Comment:
I think the function name is already too long so slightly prefer to keep it as it is 😄